src/util/RegistrationFormFactory.js

```diff
@@ -113,16 +113,31 @@ const checkSubSchemas = function(fieldName, model, subSchemas, showError) {
     } else {
       ele.children('p').addClass('default-schema');
     }
+
+    // clear role="status" on <p>
+    ele.children('p').removeAttr('role');
```

Comment:
Is there a way to remove the role in the DOM by default rather than having to do it in JS?